web/src/containers/admin/ManageAccount.js

```diff
@@ -32,6 +37,14 @@ const ManageAccount = ({
   if(error) {
     setShowConfirmation(false);
   }
+
+  const secondaryButtonText = isAdmin
+    ? 'Admin Dashboard'
+    : `< ${
```

Comment:
can you remove the <? it's not necessary here and it's inconsistent

Reply:
It has been removed